src/ducks/Alert/components/AlertModalContent/StepsContent/StepTitle/StepTitle.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "~@santiment-network/ui/mixins";
+
+.wrapper {
+  display: flex;
+}
+
+.iconWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  background: var(--waterloo);
+
+  &.disabled {
+    background: var(--porcelain);
+  }
+}
+
+.icon {
+  fill: var(--white);
+
+  &.disabled {
+    fill: var(--waterloo);
+  }
+}
+
+.titleWrapper {
+  display: flex;
+  flex-direction: column;
+  margin-left: 12px;
+
+  .title {
+    color: var(--rhino);
+
+    @include text("body-2");
+  }
+
+  .description {
+    color: var(--waterloo);
+
+    @include text("body-3");
+  }
+
+  &.disabled {
+    .title {
+      color: var(--waterloo);
+    }
+
+    .description {
+      color: var(--casper);
+    }
+  }
```

Comment:
`[D-0-1]` 
- It's not a good case to nest classes in this case: `.title`, `.description` can be moved to the outer scope;
- `.disable` selector can change the `css variables` that will affect the `.title`, `.description` colors, `.icon` fill and `.iconWrapper` background. In order for that to work, `disable` class name should be assigned to the `.wrapper` element (`LOC 8` in `/StepTitle.js`)